js/main.js

```diff
@@ -1,3 +1,5 @@
+var i;
```

Comment:
What's the global `i` for? When used in proximity of loops, it makes sense as an incrementor. However, when it's declared globally and far away from where it's used, it's not obvious what it's for, so it'd be best to either:
- If you _do_ need it available for the whole script, name it something that indicates what it's for.
- If you _don't_, declare it where it's used.